MobileWallet/Backup/ICloudBackup.swift

```diff
@@ -47,6 +48,23 @@ enum ICloudBackupWalletError: Error {
     case dbFileNotFound
     case iCloudContainerNotFound
     case unableCreateBackupFolder
+    case keychainPasswordFailture
+    case uploadToICloudFailture
+    case noInternetConnection
+}
+
+enum ICloudBackupState {
+    case upToDate
+    case outOfDate
+    case inProgress
```

Comment:
We probably will have an additional state here: `scheduled`.

Reply:
ok, just let's make sure about that and then add it